vulkan/source/gfx/bindings/vulkan/vk.d

```diff
@@ -6,13 +6,21 @@ version(Windows) {
     import core.sys.windows.windef : HINSTANCE, HWND;
 }
 
-version(linux) {
-    import xcb.xcb : xcb_connection_t, xcb_visualid_t, xcb_window_t;
+version(glfw) {
+    enum isGlfw = true;
+} else {
+    enum isGlfw = false;
 }
```

Comment:
`vk.d` is auto generated, so you should not modify it in any way.
Current short term solution is to also have wayland and xcb dependencies (even if not used) when using glfw.

If you are interested, `vk.d` is generated by [vkdgen](https://github.com/rtbo/vkdgen)
If you need to remove the dependencies, I can introduce new versions like `HasWayland` and `HasXcb` to opt them in or out at compile time.